.github/workflows/ci.yml

```diff
@@ -175,6 +172,13 @@ jobs:
       matrix:
         multiverse: [agent, api, background, background_2, database, httpclients, rails, rest, serialization, sinatra]
         ruby-version: [2.0.0-p648, 2.1.10, 2.2.10, 2.3.8, 2.4.10, 2.5.8, 2.6.6, 2.7.1, jruby-9.2.12.0]
+        include:
+          - ruby-version: "2.7.1"
+            multiverse: infinite_tracing
+          - ruby-version: "2.6.6"
+            multiverse: infinite_tracing
+          - ruby-version: "2.5.8"
+            multiverse: infinite_tracing
```

Comment:
I think since we're adding infinite tracing here, we should bump up the max_attempts for retry to 4, since that what seemed to work best for the infinite tracing runs when it was separated.

Reply:
done!